tests/unit/test_base_context.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from dbt.context.base import BaseContext
+from jinja2.runtime import Undefined
+
+
+class TestBaseContext:
+    def test_log_jinja_undefined(self):
```

Comment:
These can also be combined with `@pytest.mark.parameterize()`, though the exception message might be a bit long to do so.